setup.sh

```diff
@@ -69,6 +69,12 @@ check_dependencies() {
         exit 1
     fi
 
+    # For convenience, let's make sure github is in the known-hosts
+    # file as well.  This is a noop if it's already in there.
+    mkdir -p ~/.ssh
```

Comment:
If this directory doesn't already exist, I think there will be an issue with the permissions. Probably need:
```
mkdir -p ~/.ssh && chmod 700 ~/.ssh && touch ~/.ssh/known_hosts && chmod 600 ~/.ssh/known_hosts
```